decidim-templates/app/controllers/decidim/templates/admin/proposal_answer_templates_controller.rb

```diff
@@ -25,9 +25,9 @@ def create
           @form = form(ProposalAnswerTemplateForm).from_params(params)
 
           CreateProposalAnswerTemplate.call(@form) do
-            on(:ok) do |template|
+            on(:ok) do |_template|
```

Comment:
is is mandatory to have the do |_template| ? 
I would go for 
```
on(:ok) do 
# fadfda
```

Reply:
I guess so. I have not touched this part